README.adoc

```diff
@@ -25,13 +26,20 @@ In your YAML file, add:
 
 ----
 
-The Docker config file must be formatted in JSON, as follows:
-
+. The Docker config file must be formatted in JSON.
```

Comment:
Reviewing the README as a whole, I realized that this second prerequisite is not needed actually. The OCI credentials configuration action mentioned in the first prerequisite takes care of creating the docker config file. Instead of the second prereq, we may want to tell the user that they need to setup the property `DOCKER_REGISTRY` and secret properties `DOCKER_USERNAME` and `DOCKER_PASSWORD`.

Reply:
@urvashisingh I think because we link to the OC credentials action, we do not need to explain in detail?

Reply:
yes, that makes sense. The OC action should be explaining this in detail.